pkg/utils/utils.go

```diff
@@ -53,7 +53,7 @@ func GetPfAddr(pciAddr string) (string, error) {
 	pciinfo, err := os.Readlink(pfSymLink)
 	if err != nil {
 		if os.IsNotExist(err) {
-			return pciAddr, nil
+			return "", nil
```

Comment:
The comment for this function need to change accordingly.

Reply:
done thanks!